README.md

```diff
@@ -1,49 +1,73 @@
 # bestpy
 A module to prove your friends (or adversaries) wrong.
 
-Ever needed to decide on what is the best thing out? That's exactly what bestpy does. We may or may not try to make the answers support your view. Here's a quick demo:
+Ever needed to decide on what is the best thing out? That's exactly what bestpy does.
+We may or may not try to make the answers support your view. Here's a quick demo:
+
 ```python
 >>> best.language
 "python"
 >>> best.module
 "bestpy"
 ```
+
 ## Installation
 This is simple with pip. Just run the following in your command line or terminal (available soon):
+
 ```
 pip install bestpy
 ```
 
-You can also use your magic powers to get the module from source code. Though we won't be guiding you with that hokus-pokus here. Ask a friend if your powers aren't strong enough.
+You can also use your magic powers to get the module from source code with the following:
+
+```
+pip install git+https://github.com/gustavwilliam/bestpy.git@main
```

Comment:
Maybe we could setup a `bestpy-nightly` release, I’m sure this would be really useful.

Reply:
I haven't heard about 'nightly'. If you think it's something we should add/use/implement, feel free to open an issue about it. I'd love to hear more regarding your thoughts.